src/main/kotlin/no/skatteetaten/aurora/boober/feature/DeploymentConfigFeature.kt

```diff
@@ -50,7 +51,7 @@ val AuroraDeploymentSpec.managementPath
     }
 
 @Service
-class DeploymentConfigFeature : Feature {
+class DeploymentConfigFeature(val cantusService: CantusService) : Feature {
```

Comment:
Med tanke på førstegangslaunch av et cluster (i dagens situasjon utv05), vil avhengighet til Cantus her gjøre at også Cantus må være oppe og kjøre før man kan ta i bruk ao for å utføre deploys?

Reply:
Godt poeng. Den vil gjøre et kall til cantus for å få tak i manifestet. Hvis dette feiler så vil verdien til miljøvariabelen AURORA_HEADER_KLIENTID defaulte til deployTag og ikke den fulle versjonen fra manifestet. Jeg skal gjøre en test på dette.

Reply:
Det hadde jeg ikke tenkt på @oyvinder. Jeg fant ut at APP_VERSION settes av Architect på alle images og er tilgjengelig som miljøvariabel i podden. Det gjør at vi slipper å spørre Cantus og det vil virke for Image Change trigger. Se oppdatert kode.